src/audio/pipeline.c

```diff
@@ -215,23 +215,25 @@ int pipeline_free(struct pipeline *p)
 
 	trace_pipe_with_ids(p, "pipeline_free()");
 
+	data.start = p->source_comp;
```

Comment:
@bkokoszx this should go inside the if() below as well right? It is harmless but it is also not useful

Reply:
@ranj063 Yes, I've just updated it. Thanks